lang/java/reef-io/src/main/java/org/apache/reef/io/network/group/impl/driver/GroupCommDriverImpl.java

```diff
@@ -113,7 +113,7 @@
   @Inject
   public GroupCommDriverImpl(final ConfigurationSerializer confSerializer,
                              @Parameter(DriverIdentifier.class) final String driverId,
-                             @Parameter(TreeTopologyFanOut.class) final int fanOut) {
+                             @Parameter(TreeTopologyFanOut.class) final int fanOut) throws InjectionException {
```

Comment:
Please don't expose the `Exception` here, as it changes the signature of the constructor, which breaks current users. Instead, we'll have to catch that exception and rethrow it as a `RuntimeException`. Not idea, but better than breaking users. Also, this is temporary anyhow, right?